app/controllers/users_controller.rb

```diff
@@ -311,7 +311,7 @@ def create
       return fail_with("login.reserved_username")
     end
 
-    if user = User.where(staged: true).find_by(email: params[:email].strip.downcase)
+    if user = User.where(staged: true).find_by_email(params[:email].strip.downcase)
```

Comment:
Ah sorry my bad. So we're defining a `find_by_email` method in the `User` model. Hmm I would rename the method though cause it is confusing now. `find_by_email` used to mean that we're looking for a`User` based on the email column. Perhaps `find_by_user_email`?